wikimedia/wikipedia-ios
# Automate release tagging in Jenkins

**Fixes Phabricator ticket:** N/A

### Notes
* This takes the last beta commit, and tags it as a release and marks it in GitHub as a release.
* I had to create a new GitHub token for Jenkins to get this to run, from the wmf-apps-ci account.
* Ran `bundle update`.

### Test Steps
0. Remove existing tags: On the `wmf2249` machine, run `git tag -d releases/6.8.1 && git push --delete origin "releases/6.8.1"`
1. On the build server, run `Tag last beta as release`. When asked for a branch, put in this one: `release_tag`.
2. Confirm that there is a releases tag for 6.8.0: https://github.com/wikimedia/wikipedia-ios/tags
3. Confirm that tag is listed as the latest release: https://github.com/wikimedia/wikipedia-ios/releases

*After this gets merged, I need to remove the option in Jenkins to choose a branch - as it will always be run on `main`.*
